### PR TITLE
Fixed shared uploads from app side

### PR DESCRIPTION
**Pull request by Michal-Chatlas1221**

### CODE REVIEW

File: extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php [hunk 1]
```diff
@@ -706,6 +706,35 @@ public function execute() {
...
+		) {
+			$partnerWikiData = $dbr->selectRow(
+				array( "city_list" ),
+				array(
+					"city_id",
+					"city_url",
+					"city_dbname",
+					"city_path",
+				),
+				array( "city_list.city_dbname" => $this->mVariables[ 'wgSharedUploadDBname' ] )
```
Comment (abador) [inline extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php]: leaving only city_dbname would be enough

File: extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php [hunk 2]
```diff
@@ -706,6 +706,35 @@ public function execute() {
...
+			);
+
+			if ( !empty( $partnerWikiData ) && !( $partnerWikiData->city_path === 'slot1' ) ) {
```
Comment (abador) [inline extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php]: why not  $partnerWikiData->city_path !== 'slot1' ?:)

**Comment by abador** [comment]
Minor changes

**Review by abador — APPROVED** [review]

File: extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php [hunk 3]
```diff
@@ -706,6 +706,38 @@ public function execute() {
 			wfProfileOut( __METHOD__."-varsdb" );
 		}
 
+		#Fix shared uploads to UCP wikis
+		if (
+			!empty( $this->mVariables['wgUseSharedUploads'] ) &&
+			!empty( $this->mVariables['wgSharedUploadDBname'] ) &&
+			$this->mWikiID !== 177
```
Comment (Grunny) [inline extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php]:
```suggestion
			$this->mWikiID !== Wikia::COMMUNITY_WIKI_ID
```

File: extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php [hunk 4]
```diff
@@ -706,6 +706,38 @@ public function execute() {
...
+		) {
+			$partnerWikiData = $dbr->selectRow(
+				array( "city_list" ),
+				array(
+					"city_id",
+					"city_url",
+					"city_dbname",
+					"city_path",
+				),
+				array( "city_dbname" => $this->mVariables[ 'wgSharedUploadDBname' ] )
```
Comment (Grunny) [inline extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php]: These lines within the `selectRow` call should all use short array syntax `[]` and single quotes per coding conventions.

**Review by Grunny — APPROVED** [review]
Minor comments, otherwise looks good.

**Review by t-tomalak — APPROVED** [review]

**Review by maaartyyynaa — APPROVED** [review]

**Pull request merged**